Src/DependencyCollector/Shared/DependencyTrackingTelemetryModule.cs

```diff
@@ -56,6 +62,24 @@ public void Initialize(TelemetryConfiguration configuration)
                         {                            
                             this.telemetryConfiguration = configuration;
 
+                            if (!this.DoNotParseHttp && configuration != null)
```

Comment:
I think it is reasonable to add it by default. Let's add it by default in configuration file so it is not hidden from customers and can be removed by deleting from there rather than some undocumented property.